src/Language/PureScript/Make.hs

```diff
@@ -221,15 +221,16 @@ make MakeActions{..} ms = do
                               _ -> True
 
         let rebuild = do
-              (exts, warnings) <- listen $ do
+              (exts, warnings) <- listen $ censor suppressHoleWarnings $ do
```

Comment:
@paf31 attempting to censor the warnings here (to knock out the hole appearing as both warning and error) has no effect. In fact, using `mempty` instead of `warnings` down at `markComplete` seems to have no effect either. Do you know where it is that they're being printed from? And is there any point to `listen`ing to the warnings here if they're as good as unused?